File: app/src/main/java/com/wkz/viewpager/transformer/DefaultTransformer.java
package com.wkz.viewpager.transformer;

import android.view.View;

/**
 * @desc 默认转换效果
 * @date 2017-06-30
 */
public class DefaultTransformer extends AbstractBaseTransformer {
    @Override
    protected void onTransform(View page, float position) {

    }

    @Override
    protected boolean isPagingEnabled() {
        return true;
    }
}

File: view-pager-transformer/src/main/java/com/fphoenixcorneae/transformer/DefaultTransformer.kt
package com.fphoenixcorneae.transformer

import android.view.View

/**
 * @desc 默认转换效果
 * @date 2017-06-30
 */
class DefaultTransformer : AbstractBaseTransformer() {

    override val isPagingEnabled: Boolean = true

    override fun onTransform(page: View, position: Float) {}
}